### ISSUE TITLE
dataset passed to PythonScriptStep has no name or version

### ISSUE THREAD

**Issue by swanderz** [Azure/MachineLearningNotebooks#669]
SDK version: `1.0.69`
#### pipeline extract
```python
    dataset_version = params['dataset_version']

    dataset_raw = Dataset.get_by_name(workspace=ws,
                                      name='deal_raw',
                                      version=dataset_version)

    extract_step = PythonScriptStep(
        name='extract',
        script_name='extract.py',
        arguments=['--output_dir', extracted_data,
                   '--remote_run', True,
                #    '--n_rows', 10000
                   ],
        compute_target=compute_target_big,
        inputs=[dataset_raw.as_named_input('dataset_raw')],
        outputs=[extracted_data],
        runconfig=run_config,
        source_directory=os.path.join(os.getcwd(), 'compute', 'extract'),
        allow_reuse=pipeline_reuse
    )
```

#### `extract.py` extract
```python
dataset = run.input_datasets['dataset_raw']
print("Using {} dataset version {}".format(dataset.name, dataset.version))
```

#### `70_driver_log.txt`
```python
Using None dataset version None
```

**Comment by YutongTie-MSFT** [comment]
@swanderz 
Thanks for the feedback! We are currently investigating and will update you shortly.

@danielsc 
Hi Daniel, could you please help us check this issue? Thanks.

**Comment by shuyums2** [comment]
The behavior is fixed in SDK `1.0.72`.

If you update the conda spec for remote env to install SDK version higer than `1.0.72`, `name` and `version` should not be None anymore. (at the moment of this reply the latest SDK version is `1.0.74`)

**Comment by YutongTie-MSFT** [comment]
@swanderz 
Thanks for reaching out to us. This has been fixed in the last version of SDK. We will now proceed to close this thread. If there are further questions regarding this matter, please respond here and @YutongTie-MSFT and we will gladly continue the discussion.
